spec/features/homepage_spec.rb

```diff
@@ -1,8 +1,19 @@
 require 'rails_helper'
 
 feature 'Homepage' do
-  scenario 'see "Smartflix"' do
+  scenario 'see "Smartflix" as the main header' do
     visit '/'
-    expect(page).to have_content 'Smartflix'
+    expect(page).to have_selector('h1', text: 'Smartflix')
+  end
+
+  scenario 'see the titles of the first six shows' do
+    visit '/'
+    expect(page).to have_selector('h3', text: 'Dick Johnson Is Dead 2020')
+    expect(page).to have_selector('h3', text: 'Blood & Water 2021')
+    expect(page).to have_selector('h3', text: 'Ganglands 2021')
+    expect(page).to have_selector('h3', text: 'Jailbirds New Orleans 2021')
+    expect(page).to have_selector('h3', text: 'Kota Factory 2021')
+    expect(page).to have_selector('h3', text: 'Midnight Mass 2021')
+    expect(page).not_to have_selector('h3', text: 'My Little Pony: A New Generation 2021')
```

Comment:
Nice thorough test! It may suffice to match on text, rather than the selector specifically. For e2e tests, I try to have the test know as little about the structure of the code (and here HTML is structure) as possible.